packages/UI/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  ChevronsLeft,
+  ChevronsRight,
+  ChevronRight,
+  ChevronLeft,
+} from 'lucide-react';
+import Link from 'next/link';
+import * as React from 'react';
+
+import type { VariantProps } from 'tailwind-variants';
+import { tv } from 'tailwind-variants';
+import { cn } from '../../utils/classesMerger';
```

Comment:
What is the benefit from having twMerge in this utils file? Seems like just an extra file/import just to implement twMerge as a different file.

Reply:
`cn` does the same thing twMerge but does a bit more by enabling objects to be passed over.
for example let's say some is using the `<Pagination />` and wants a specific style applied only on a specific condition: `<Pagination className={{'bg-black': someCondition}}>...</Pagination>`.
this is only possible using `cn` but if we use twMerge they can't pass object with condition.